model/Capability.ts

```diff
@@ -0,0 +1,6 @@
+export interface Capability
+{
+    capabilityId?: number
+    name?: string
+    description?: string
+}
```

Comment:
Is there a scenario where the API doesn't send the these values back? 

```suggestion
export interface Capability
{
    capabilityId: number
    name: string
    description: string
}
```